dashboard/src/components/partials/ConfirmDelete/Dialog.js

```diff
@@ -45,7 +45,7 @@ class ConfirmDeleteDialog extends React.Component {
         <DialogTitle id={titleId}>Confirm</DialogTitle>
         <DialogContent>
           <Typography>
-            Are you sure you would like to permanantly delete{" "}
+            Are you sure you would like to permenantly delete{" "}
```

Comment:
🤦‍♂️

Reply:
I spelled it wrong there, whoops.

Reply:
But if you're facepalming there was a different but similar line you could do that on ;)